bika/lims/exportimport/instruments/instrument.pt

```diff
@@ -4,6 +4,8 @@
 <label for='instrument_results_file_format'>Format</label>&nbsp;
 <select name="instrument_results_file_format" id="instrument_results_file_format">
     <option value='csv'>CSV</option>
+    <option value='xls'>XLS</option>
+    <option value='xlsx'>XLSX</option>
```

Comment:
As far as I know, this is the "default" instrument import view, so these options might depend on the available parsers for a given instrument. Maybe one instrument has parsers for all three formats, but other instruments will only have parsers for just one (e.g. CSV). Or, are you handling all them generically?

Reply:
You are correct, I could define a specific a local view for relevant instruments.  Another question I have is what benefit does the Format field have at all? Are the some instruments that allow more that one format?

Reply:
Some instruments are capable to generate results files in different formats (e.g. XML + CSV), so the idea was the keep same form view for a given instrument, even if it's capable to generate results files in different formats.

Reply:
Cool. I'll revert this and create local templates.

Reply:
Removed commit from branch and updated PR